chemprop/uncertainty/evaluator.py

```diff
@@ -67,9 +160,33 @@ def evaluate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -
 
 @UncertaintyEvaluatorRegistry.register("spearman")
 class SpearmanEvaluator(UncertaintyEvaluator):
+    """
+    Evaluate the Spearman rank correlation coefficient between the uncertainties and errors in the model predictions.
+
+    The correlation coefficient returns a value in the [-1, 1] range, with better scores closer to 1
+    observed when the uncertainty values are predictive of the rank ordering of the errors in the model prediction.
+
+    References
+    ----------
+    .. [spearman1904] Spearman, C. "The Proof and Measurement of Association between Two Things."
+    Am. J. Psychology. 1904, 15 (1), 72-101.
```

Comment:
does this need an explanation? Also, not that the reference is a bad thing, but I'm surprised you actually went the trouble of finding it! I feel like once something get's a name it's almost self-referential

Reply:
I think it would be fine without the explanation. I essentially copied over the v1 docstring just to be safe. As for the reference, it's also probably not necessary like you say. I went to torchmetrics' documentation for Spearman, and they refer to Wikipedia, which actually has the original reference. So it wound up being easier to find than perhaps expected. I'm happy to keep or remove it as we see fit, and I'll implement the other formatting changes above too.